manifests/base_apps.pp

```diff
@@ -93,12 +93,19 @@
 
         # portreserve will only start if there is a file in the conf directory
         # such as: cups ipp dhcpd named slapd ldaps
+        file { '/etc/portreserve/sshd':
```

Comment:
@jeefberkey This needs to be here for standalone to work *at all* due to the removal of Puppet at the end. It's harmless.